rust/processor/src/db/postgres/migrations/2025-01-08-171729_emojicoin_arena/up.sql

```diff
@@ -125,17 +137,38 @@ CREATE TABLE arena_info (
     melee_id NUMERIC NOT NULL PRIMARY KEY,
     volume NUMERIC NOT NULL,
     rewards_remaining NUMERIC NOT NULL,
-    apt_locked NUMERIC NOT NULL,
+    emojicoin_0_locked NUMERIC NOT NULL,
+    emojicoin_1_locked NUMERIC NOT NULL,
 
     -- Redundant information to avoid multiple queries/joins
     emojicoin_0_market_address TEXT,
     emojicoin_1_market_address TEXT,
+    emojicoin_0_symbols TEXT[],
+    emojicoin_1_symbols TEXT[],
+    emojicoin_0_market_id NUMERIC,
+    emojicoin_1_market_id NUMERIC,
     start_time TIMESTAMP,
     duration NUMERIC,
     max_match_percentage NUMERIC,
     max_match_amount NUMERIC
 );
 
+CREATE TABLE arena_candlestick (
+    melee_id NUMERIC NOT NULL,
+
+    period period_type NOT NULL,
+    start_time TIMESTAMP NOT NULL,
+
+    open_price NUMERIC,
+    high_price NUMERIC,
+    low_price NUMERIC,
+    close_price NUMERIC,
+    volume NUMERIC NOT NULL,
+    n_swaps NUMERIC NOT NULL,
```

Comment:
```suggestion
    open_price NUMERIC,
    high_price NUMERIC,
    low_price NUMERIC,
    close_price NUMERIC,
    volume NUMERIC NOT NULL,
    n_swaps NUMERIC NOT NULL,
    last_version BIGINT NOT NULL,
```

The latest candlestick needs to have enough information for the client for it to be able to build off of it. `n_swaps` doesn't really indicate where in time it stopped recording (for the current candlestick), but I think `version` is just way simpler, it's more obvious